pkg/portal/graphql/app_mutation.go

```diff
@@ -31,6 +31,10 @@ var appResourceUpdate = graphql.NewInputObject(graphql.InputObjectConfig{
 			Type:        graphql.String,
 			Description: "New data of the resource file. Set to null to remove it.",
 		},
+		"checksum": &graphql.InputObjectFieldConfig{
+			Type:        graphql.String,
+			Description: "Checksum of the resource file. Set to null to remove it.",
```

Comment:
The "Set to null to remove it" is incorrect.

Let's use this

> The checksum of the original resource file. If provided, it will be used to detect conflict.